src/transport/SecureSessionMgr.cpp

```diff
@@ -257,8 +257,8 @@ CHIP_ERROR SecureSessionMgr::NewPairing(const Optional<Transport::PeerAddress> &
             state, [this](const Transport::PeerConnectionState & state1) { HandleConnectionExpired(state1); });
     }
 
-    ChipLogDetail(Inet, "New pairing for device 0x%08" PRIx32 "%08" PRIx32 ", key %d!!", static_cast<uint32_t>(peerNodeId >> 32),
-                  static_cast<uint32_t>(peerNodeId), peerKeyId);
+    ChipLogDetail(Inet, "New secure session created for device 0x%08" PRIx32 "%08" PRIx32 ", key %d!!",
+                  static_cast<uint32_t>(peerNodeId >> 32), static_cast<uint32_t>(peerNodeId), peerKeyId);
```

Comment:
This can use `ChipLogFormatX64` and `ChipLogValueX64`, right?

Reply:
Yes, didn't realize these formatter existed. Updated the code.